error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestOutput.java

```diff
@@ -137,4 +138,28 @@ ImmutableSet<Boolean> testStreamAllMatch() {
   boolean testStreamAllMatch2() {
     return Stream.of("foo").allMatch(s -> s.isBlank());
   }
+
+  ImmutableSet<Integer> testStreamMapToIntSum() {
+    Function<String, Integer> parseIntFunction = Integer::parseInt;
+    return ImmutableSet.of(
+        Stream.of(1).mapToInt(i -> i * 2).sum(),
+        Stream.of("1").mapToInt(Integer::parseInt).sum(),
+        Stream.of("1").map(parseIntFunction).reduce(0, Integer::sum));
+  }
+
+  ImmutableSet<Double> testStreamMapToDoubleSum() {
+    Function<String, Double> parseDoubleFunction = Double::parseDouble;
+    return ImmutableSet.of(
+        Stream.of(1).mapToDouble(i -> i * 2.0).sum(),
+        Stream.of("1").mapToDouble(Double::parseDouble).sum(),
```

Comment:
We generally use a different number or string per test case, see also further up in this file :). So let's change this.